crates/host/src/wasmbus/mod.rs

```diff
@@ -349,42 +355,29 @@ impl Drop for Provider {
 
 /// wasmCloud Host
 pub struct Host {
-    components: RwLock<HashMap<ComponentId, Arc<Component>>>,
     event_builder: EventBuilderV10,
     friendly_name: String,
-    heartbeat: AbortHandle,
+    heartbeats: Vec<AbortHandle>,
     host_config: HostConfig,
     host_key: Arc<KeyPair>,
+    #[allow(unused)]
```

Comment:
is this leftover from avoiding clippy or do we still need the 3 `#[allow(unused)]` macros in the Host struct

Reply:
It's for a follow-up PR where we will want these values again when we have hosts that need to listen for traffic for multiple lattices. I didn't want to remove them since they're just going to come back